### PR TITLE
Add Ticker sample

### PR DESCRIPTION
**Pull request by stuartmscott**

### CODE REVIEW

File: samples/time/docker-compose.yml [hunk 1]
```diff
@@ -34,6 +35,7 @@ services:
         sleep 1
         ./build/src/Wink send 10.0.0.2:64002 stop
         ./build/src/Wink send 10.0.0.2:64002 exit
+        ./build/src/Wink start -a 10.0.0.4:42001 time/Ticker 10.0.0.2:64003 2
```
Comment (winksaville) [inline samples/time/docker-compose.yml:38]: Is this an actual test, I see you're executing the Ticker but I don't see where you're "testing" that it's operating correctly?

**Review by winksaville — DISMISSED** [review]

File: samples/time/docker-compose.yml (hunk 1 above)
Comment (stuartmscott) [reply samples/time/docker-compose.yml:38]: Its a manual test - I brought up this docker compose and saw in the logs the Ticker had sent ticks;

Screen Shot 2023-01-15 at 09 18 32

Comment (stuartmscott) [reply samples/time/docker-compose.yml:38]: (note: the server receives the `start` message before the client sends it - this is just an artifact of the logs from two containers getting merged)

Comment (winksaville) [reply samples/time/docker-compose.yml]: Ordering is very important "between" sender/receiver pairs, but this is a perfect example of the ordering "across" sender/receiver pairs cannot be guaranteed :)

Also a "partial" solution I've provided for my Rust code is a [custom logger](https://github.com/winksaville/custom-logger/blob/main/src/lib.rs#L9) which adds "[timestamp in nanos, LogLevel, module_path, line number and threadId] for every message:
```
73 contents_se='[2022-05-07T16:19:03.847496514Z INFO  custom_logger::test   54  3] hello
```

**Review by winksaville — APPROVED** [review]

**Pull request merged**